05.NablaTutorialExample/main.cpp

```diff
@@ -60,7 +60,7 @@ class NablaTutorialExampleApp : public ApplicationBase
 	std::array<nbl::video::IGPUQueue*, CommonAPI::InitOutput::MaxQueuesCount> queues = { nullptr, nullptr, nullptr, nullptr };
 	nbl::core::smart_refctd_ptr<nbl::video::ISwapchain> swapchain;
 	nbl::core::smart_refctd_ptr<nbl::video::IGPURenderpass> renderpass;
-	std::array<nbl::core::smart_refctd_ptr<nbl::video::IGPUFramebuffer>, CommonAPI::InitOutput::MaxSwapChainImageCount> fbo;
+	nbl::core::smart_refctd_dynamic_array<nbl::core::smart_refctd_ptr<nbl::video::IGPUFramebuffer>> fbo;
```

Comment:
Is there any reason why you didn't do this change for Example 02? 
Also What was the motivation to use dynamic array instead of std::array with MaxSwapChainImages?

Reply:
nvm I figured it out